test/LaunchDarkly.EventSource.Tests/EventSourceEndToEndTest.cs

```diff
@@ -137,20 +115,16 @@ public void ReadTimeoutIsDetectedInRawUtf8ReadingMode()
             }
         }
 
-        private static void WriteChunk(Stream stream, string data)
-        {
-            var bytes = Encoding.UTF8.GetBytes(data);
-            stream.Write(bytes, 0, bytes.Length);
-            stream.Flush();
-        }
-
         private static WebServer StartWebServerOnAvailablePort(out Uri serverUri, Action<IHttpContext> handler)
         {
             var module = new SimpleModule(handler);
 
             for (int port = 10000; ; port++)
             {
-                var server = new WebServer(port).WithModule(module);
+                var options = new WebServerOptions()
+                    .WithUrlPrefix($"http://*:{port}")
+                    .WithMode(HttpListenerMode.EmbedIO);
```

Comment:
EmbedIO can run in two modes: the EmbedIO mode, where it always uses its own web server implementation, or a mode where it uses some built-in Windows stuff if that is available. Using the former makes the behavior more predictable cross-platform.